pkgs/by-name/ku/kubectl-kcl/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-kcl";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = pname;
```

repo = pname is an anti pattern
```suggestion
    repo = "kubectl-kcl";
```

---

amended